### ISSUE TITLE
TurboParser has no "--server" option

### ISSUE THREAD

**Issue by alexandrutodor** [sammthomson/semviz#5]
Hi,

Services.py is calling TurboParser with a "--server" option, however TurboParser gives the folowing error:

> ERROR: unknown command line flag 'server'

I couldn't find any reference to the server option in any version of TurboParser. Are you using a custom version, or am I just doing something wrong.

I've configured everything but can't seem to get past this error. I'd appreciate any tip you can give me.

**Comment by sammthomson** [comment]
Yes, unfortunately, it's a custom version of TurboParser that I thought would eventually get released but never did. You could try using [MaltParser](http://www.maltparser.org/mco/english_parser/engmalt.html) and `services.MaltClient` instead of `services.TurboClient`. SEMAFOR's accuracy is almost identical.
